packages/slice-machine/src/theme.ts

```diff
@@ -21,13 +21,21 @@ const AppTheme = (): Theme =>
           color: "#27AE60",
         },
         new: {
-          bg: "#F6F1FC",
-          color: "#AC7EE9",
+          bg: "#C2EFC899",
+          color: "#32A943",
         },
         modified: {
           bg: "#FDF4EC",
           color: "#F2994A",
         },
+        synced: {
```

Comment:
oh weird I though we already had the synced one

Reply:
yup! there wasn't any theme applied for synced haha